src/app/home/home.component.html

```diff
@@ -69,10 +69,27 @@ <h5>${{getProgress()}} of ${{getGoal()}} donated.</h5>
         </div>
     </mat-sidenav>
 
+
     <!--HOME PAGE CONTENT GOES HERE-->
     <mat-sidenav-content class="main-content-container">
         <div class="main-content">
             <div class="payment-feed-container">
```

Comment:
add styles to this container to center each card

Reply:
something like display: flex
flex direction column
justify content and align items center